CIP-1904/README.md

```diff
@@ -0,0 +1,250 @@
+As we are working with off-chain JSON, every signature must be a [JSON Web Signature](https://datatracker.ietf.org/doc/html/rfc7515) (JWS - RFC-7515) and the JWS header must be included in the transaction metadata.
+
+A single hash is taken of the entire JSON batch and must be represented as a [Content Identifier](https://github.com/multiformats/cid) (CID).
+Usage of CIDs allow us to represent the hash in a self-describing way that is not ambiguous.
+The subtype `st` may limit the types of CIDs used in order to not require 3rd party verification applications to support too many hash algorithms.
+
+The CID itself, which is embedded in the metadata of the transaction is used as an identifier for the batch and hence we recommend using this as a unique identifier to resolve the off-chain data.
+
+### Transaction types
+Currently, there are 3 different types of verifiable supply chain records supported:
+- Supply Chain Management data,
+- Certificate of Conformity issue,
+- Certificate of Conformity revoke
+
+The [CDDL schema](./schema.cddl) covers all transaction types.
+
+#### Supply Chain Management data
+Supply Chain Management (SCM) data is associated with a good or product as a result of production, manufacturing and transport processes throughout its lifecycle.
+This data may be shared between all supply chain stakeholders as well as delivered to end customers.
+A producer is responsible to collect the SCM data associated with its product/good and insert it into the supply chain ecosystem in a defined manner specific to the use-case.
+
+We strongly recommend that the off-chain SCM schema follows an industry standard relevant to the particular use-case.
+For example, the Georgian Wine use-case from the Cardano Foundation with the National Wine Agency of Georgia uses an extended version of the [OIV Standard](https://www.oiv.int/standards/international-standard-for-the-labelling-of-wines) from the [International Organisation of Vine and Wine](https://www.oiv.int).
+Using such standards brings credibility to the transactions and helps foster enterprise adoption.
+
+##### Off-chain format
+Each verifiable record comes from a specific producer by `producerID`.
+An SCM batch can contain records from multiple producers, hence the off-chain data is a JSON map of `producerID` to an array of JSON objects.
+The structure of the JSON objects within the array is dependant on the subtype `st`.
+
+The `producerID` is a unique identifier for a producer in the supply chain ecosystem of the given subtype.
+We recommend short IDs if possible to save on space and costs.
+
+The following example contains 3 verifiable records from producer `a2gh` and producer `p33l`, where `{...}` represents an arbitrary JSON object for a given verifiable record:
+```
+{
+  a2gh: [{...}, {...}, {...}],
+  p33l: [{...}, {...}]
+}
+```
+
+##### On-chain format
+The following field labels are used within the on-chain metadata:
+
+|**Field**|**Description**|**Type**|**Set Value**|
```

Comment:
I only put the table in once since it the others use a subset of these fields. But maybe I should move this to higher up before the SCM section to be more "common" and just be a master list of supported fields?

The other "subtypes" generally should just dictate off-chain and how to resolve keys/data etc, and not impact this structure.

Really there's just 2 formats here: batching with sigs from multiple parties (wineries SCM), and batching with sigs from a single party (NWA certs).